fix: App update with no permission causes invalid_installation_disabled (#35594)
RocketChat/Rocket.Chat: .changeset/shy-moles-boil.md

---
"@rocket.chat/meteor": patch
---

> fixes an issue where removing the permissions of an app would cause it to never be enabled.

Reviewer: ```suggestion
Fixes an issue where removing the permissions of an app would make it impossible to enable it after update
```

> fixes an issue where removing the permissions of an app would cause it to never be enabled.

Reviewer: @kody changeset files are not really meant to be comprehensive documentation, they should be short and concise.
